conda-forge.yml

```diff
@@ -1,6 +1,3 @@
-bot:
-  abi_migrtation_branches:
-  - java11
```

Comment:
The linter raised an error on this. I removed it but I'm not sure why it was still there.